integration_tests/e2e/countersign.cy.ts

```diff
@@ -26,3 +28,33 @@ describe('View Plan Overview for READ_ONLY user', () => {
     cy.checkAccessibility()
   })
 })
+
+describe('View specific plan version for READ_ONLY user', () => {
+  let pk
+
+  beforeEach(() => {
+    cy.createSentencePlan().then(planDetails => {
+      cy.wrap(planDetails).as('plan')
+      pk = planDetails.oasysAssessmentPk
+      cy.addGoalToPlan(planDetails.plan.uuid, DataGenerator.generateGoal()).then(goal => {
+        cy.addStepToGoal(goal.uuid, DataGenerator.generateStep())
+      })
+      cy.lockPlan(planDetails.plan.uuid)
+    })
+  })
+
+  it('Should have one button', () => {
```

Comment:
Think there's more work to do here, tests nothing about buttons.

Reply:
finished this as much as I can right now in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/206/commits/5d7fe1e6e2b9300569a9695c5e767ccf1dfc8554 - ideally I think it would sign out and then sign back in again to check the list twice but our sign-out doesn't quite work right so trusting that `addGoalToPlan` is I think as much as I can do now!